spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/neo4j/Neo4jProperties.java

```diff
@@ -43,23 +45,20 @@
 	static final String DEFAULT_BOLT_URI = "bolt://localhost:7687";
 
 	static final String BOLT_DRIVER = "org.neo4j.ogm.drivers.bolt.driver.BoltDriver";
-
+	private final Embedded embedded = new Embedded();
 	/**
 	 * URI used by the driver. Auto-detected by default.
 	 */
 	private String uri;
-
 	/**
 	 * Login user of the server.
 	 */
 	private String username;
-
 	/**
 	 * Login password of the server.
 	 */
 	private String password;
-
-	private final Embedded embedded = new Embedded();
```

Comment:
There is no reason to move `Embedded` at the top. Please move it back where it was.